_includes/v2.1/prod-deployment/secure-start-nodes.md

```diff
@@ -129,15 +129,14 @@ For each initial node of your cluster, complete the following steps:
     {% include {{ page.version.version }}/prod-deployment/securecockroachdb.service %}
     ~~~
 
-    Save the file in the `/etc/systemd/system/` directory.
-
 9.  Customize the sample configuration template for your deployment:
 
     Specify values for the following flags in the sample configuration template:
 
      Flag | Description
      -----|------------
      `--join` | Identifies the address and port of 3-5 of the initial nodes of the cluster.
+     `--advertise-host` | The hostname or IP address to advertise to other CockroachDB nodes. If it is a hostname, it must be resolvable from all nodes; if it is an IP address, it must be routable from all nodes.
```

Comment:
I'm in the process of updating this and other text around --advertise-host in a separate PR. I can just resolve the conflict on my end, or you can revert this change. Your call.